doc/libvips-arithmetic.md

```diff
@@ -65,7 +65,7 @@ generally the principle is that the output type should be large enough to
 represent the whole range of possible values, except that int never becomes
 float.
 
-## Arithmetic functions
+## Functions
 
 * [method@Image.add]
```

Comment:
We are setting ourselves up for a lot of work if these lists are going to be maintained by hand :(

Reply:
I'll check if we can automate this. Perhaps we could inspect the resulting GIR file and extract only the functions from the `arithmetic/` directory.

Reply:
Commit 27016bb23cb4a1acf7e0b5404e399d57b8c24d41 adds a sanity check script to verify that all symbols are listed on the relevant pages. It caught a few overlooked symbols, which were fixed separately in commit fa378cd0bc98291904433404cd20275b49823258. Linux CI was modified to cover this, see commit 6122324dbbec6e79db30063bbba08b3f76e54fc5 and e.g. these now fixed failures:
https://github.com/libvips/libvips/actions/runs/14282791823/job/40034288723#step:10:48

Reply:
Ah very nice!